src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let strArray = sourceString.replace(/[;]/g, '').split('\n');
+  const result = {};
+
+  strArray = strArray
+    .filter(x => (x.length > 0) && (x !== '  '))
+    .map(x => x.split(':'));
```

Comment:
It's never a good idea to use such kind of checks `x !== '  '`. What if there would be 3 spaces? Or 4? Then your code would not work. You can cut all unnecessary spaces in such way:
```suggestion
  strArray = strArray
    .map(x => x.trim())
    .filter(x => x.length > 0)
    .map(x => x.split(':'));
```

Reply:
done